2.6/get-pip.py

```diff
@@ -135,7 +135,7 @@ def parse_args(self, args):
     for arg in args:
         try:
             req = InstallRequirement.from_line(arg)
-        except:
+        except Exception:
```

Comment:
@hugovk the set of linter fixes has been moved to #30 as requested by @pfmoore. Feel free to drop them from here.

Reply:
Done!